src/NuGet.Core/NuGet.ProjectManagement/Projects/BuildIntegratedNuGetProject.cs

```diff
@@ -54,6 +49,22 @@ public abstract class BuildIntegratedNuGetProject : NuGetProject, INuGetIntegrat
             INuGetProjectContext projectContext,
             bool throwOnFailure);
 
+        public abstract Task<bool> InstallPackageAsync(
+            PackageIdentity packageIdentity,
+            INuGetProjectContext nuGetProjectContext,
+            IEnumerable<NuGetFramework> successfulFrameworks,
+            IEnumerable<NuGetFramework> unsucessfulFrameworks,
```

Comment:
Actually, I thought about this some more and I don't think this overload would add any value. Current P.J. and legacy implementations ignore these parameters. I think this is actually preferable than forking the code to call the two different overloads.